### DISCUSSION TITLE
Document uprating parameters

### DISCUSSION THREAD

**Discussion by nikhilwoodruff** [PSLmodels/tax-microdata-benchmarking#263] [General]
We currently use:

* Population growth by Census: [CBO: The Demographic Outlook: 2024 to 2054](https://www.cbo.gov/publication/59697) (Figure 2)
* CBO's AGI and income by source projections: [Revenue Projections, by Category | Congressional Budget Office | Feb 2024](https://www.cbo.gov/system/files/2024-02/51138-2024-02-Revenue.xlsx)
* IRS's total among filer values from [IRS SOI Tax Stats - Individual Statistical Tables by Size of Adjusted Gross Income](https://www.irs.gov/statistics/soi-tax-stats-individual-statistical-tables-by-size-of-adjusted-gross-income)

The part in the codebase that applies the uprating factors is [here](https://github.com/PSLmodels/tax-microdata-benchmarking/blob/a81ea8a6ca05008cda1600aaed804848718bacfe/tax_microdata_benchmarking/create_flat_file.py#L708).

PolicyEngine US code [here](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/calibration/gov/irs/soi/alimony_income.yaml).

**Comment by martinholmer** [comment]
@nikhilwoodruff, you assigned fixing issue #54 to yourself on April 23rd.   Where do you stand on resolving this issue?
